test/unit/specs/core/network/createNetwork.spec.js

```diff
@@ -101,35 +101,28 @@ describe("createNetwork", () => {
     });
   });
 
-  it("runs onResponseError hook and rejects the returned promise", () => {
-    networkStrategy.and.returnValue(Promise.reject(new Error("myerror")));
-    return network
-      .sendRequest({}, true)
-      .then(() => {
-        throw Error("Expecting sendRequest to fail.");
```

Comment:
I'm not sure why we were throwing an error here...

Reply:
If sendRequest resolves without error, then this test will pass.  Said another way, the expects only run when sendRequest fails.

Reply:
Ah, that makes a lot of sense. Thanks for explaining. I've added it back.